homeassistant/components/wink.py

```diff
@@ -97,9 +97,16 @@ def setup(hass, config):
     hass.data[DOMAIN] = {}
     hass.data[DOMAIN]['entities'] = []
     hass.data[DOMAIN]['unique_ids'] = []
+
+    def keep_alive_call(event):
+        """Call the Wink API endpoints to keep PubNub working."""
+        pywink.wink_api_fetch()
+        pywink.get_user()
+    hass.services.register(DOMAIN, 'keep_alive', keep_alive_call)
```

Comment:
Why would we expose this to the user? This seems to me like an implementation detail that we should just do instead of allowing the user to do it.